modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/test/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/FutureDatesFunctionTest.java

```diff
@@ -38,14 +39,28 @@ public void testApplyFalse() {
 	@Test
 	public void testApplyTrue() {
 		LocalDate tomorrowLocalDate = _todayLocalDate.plusDays(1);
+		LocalDateTime tomorrowLocalDateTime = _todayLocalDateTime.plusDays(1);
+
+		Assert.assertTrue(
+			_futureDatesFunction.apply(
+				_todayLocalDate.toString(), _todayLocalDate.toString()));
```

Comment:
This is the [original class](https://github.com/liferay/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/test/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/FutureDatesFunctionTest.java).
The Assert function was sorted by the parameters. Public variable first than the private one: tomorrowLocalDate > _todayLocalDate (even though they do not follow alphabetical order). I believe that it is public > private, then we sort by alphabetical order.

So, could you move this code to the end please?